webapp/src/components/UserBubble.vue

```diff
@@ -1,21 +1,33 @@
 <template>
-  <img
-    :src="
-      'https://www.gravatar.com/avatar/' +
-      md5(this.creator.contact_email || this.creator.display_name) +
-      '?d=' +
-      this.gravatar_style
-    "
-    class="avatar"
-    :width="size"
-    :height="size"
-    :title="this.creator.display_name"
-  />
+  <div class="bubble-wrapper">
```

Comment:
I think we should make a separate component for the top bar "ThisUserBubble" vs the one used in the creators lists, that way we can also have custom components for admin vs normal user